ui/dashboards/src/components/PanelDrawer/PanelEditorForm.tsx

```diff
@@ -112,162 +119,165 @@ export function PanelEditorForm(props: PanelEditorFormProps): ReactElement {
   const watchedName = useWatch({ control: form.control, name: 'panelDefinition.spec.display.name' });
   const watchedDescription = useWatch({ control: form.control, name: 'panelDefinition.spec.display.description' });
   const watchedPluginKind = useWatch({ control: form.control, name: 'panelDefinition.spec.plugin.kind' });
+  const { timeRange } = useTimeRangeParams(initialTimeRange);
```

Comment:
What do you think about the following suggestion?

```suggestion
  const { timeRange } = useTimeRange();
```

this would re-use the same time range which the user choose.

Reply:
on the other hand, maybe using the default time range, as you have it currently, is better. approving this PR :)